### DISCUSSION TITLE
Build image manually failing when chroot

### DISCUSSION THREAD

**Discussion by maxymo** [johang/sd-card-images#91] [General]
I am trying to build manually image for Debian Bookwork for arm64 (ODROID HC4) following the steps in this repo but when building the image get the error below, any idea?

```
+ chroot debian /bin/sh -ex
chroot: failed to run command '/bin/sh': Exec format error
```

The pre-build image works but unfortunately the size of the root partition is only 3GB and can't install proxmox backup server in it. I was hoping to be able to customise the size of the ext4 partition if I build it manually because not sure I can resize the / partition

**Comment by johang** [comment]
You probably need `qemu-user-static`.

But you can extend the root filesystem on the pre-built images: https://sd-card-images.johang.se/tips-and-tricks.html

**Comment by mdurkowski** [comment]
If you got this error on arch linux, you will also need `qemu-user-static-binfmt` package.
